descriptors/ModuleDescriptor-template.json

```
@@ -481,7 +497,9 @@
         "circulation-storage.cancellation-reasons.item.post",
         "circulation-storage.cancellation-reasons.item.put",
         "circulation-storage.cancellation-reasons.collection.delete",
-        "circulation-storage.cancellation-reasons.item.delete"
+        "circulation-storage.cancellation-reasons.item.delete",
+        "circulation-storage.patron-notice-policies.item.create",
```

This naming is inconsistent with the current convention of using post as the final postfix in the permission name for a POST endpoint. Please can this be changed to fit with the current convention.
```suggestion
        "circulation-storage.patron-notice-policies.item.post",
```

(admittedly this convention is a little inconsistent as it uses item for collection level posts)

---

changed to post

---

actually it's item level post, there is no possibility to post a collection